virt.te

```diff
@@ -304,6 +309,62 @@ files_type(virt_qemu_ga_data_t)
 type virt_qemu_ga_unconfined_exec_t, virt_file_type;
 application_executable_file(virt_qemu_ga_unconfined_exec_t)
 
+# virtinterfaced
+virt_driver_template(virtinterfaced)
+
+type virtinterfaced_var_run_t;
+files_pid_file(virtinterfaced_var_run_t)
+
+# virtnetworkd
+virt_driver_template(virtnetworkd)
+
+type virtnetworkd_var_run_t;
+files_pid_file(virtnetworkd_var_run_t)
+
+# virtnodedevd
+virt_driver_template(virtnodedevd)
+
+type virtnodedevd_var_run_t;
+files_pid_file(virtnodedevd_var_run_t)
+
+# virtnwfilterd
+virt_driver_template(virtnwfilterd)
+
+type virtnwfilterd_var_run_t;
+files_pid_file(virtnwfilterd_var_run_t)
+
+# virtproxyd
+virt_driver_template(virtproxyd)
+
+type virtproxyd_var_run_t;
+files_pid_file(virtproxyd_var_run_t)
+
+# virtqemud
+virt_driver_template(virtqemud)
+
+# virtsecretd
+virt_driver_template(virtsecretd)
+
+type virtsecretd_var_run_t;
+files_pid_file(virtsecretd_var_run_t)
+
+# virtstoraged
+virt_driver_template(virtstoraged)
+
+type virtstoraged_var_run_t;
+files_pid_file(virtstoraged_var_run_t)
+
+# virtvboxd
+virt_driver_template(virtvboxd)
+
+type virtvboxd_var_run_t;
+files_pid_file(virtvboxd_var_run_t)
+
+#virtxend
+type virtxend_t;
+type virtxend_exec_t;
+init_daemon_domain(virtxend_t, virtxend_exec_t)
```

Comment:
Why virt_driver_template is not used here? Is it for purpose?

Reply:
I skipped policy for virtxend driver for now.